Author: Stefano Zacchiroli <zack@debian.org>
Description: fix a typo in the findlib package version number specified in META
--- ulex0.8.orig/META
+++ ulex0.8/META
@@ -1,4 +1,4 @@
-version = "0.7"
+version = "0.8"
 requires = "camlp4"
 description = "Runtime support for ulex"
 archive(byte) = "ulexing.cma"
